doc/cephfs/scrub.rst

```diff
@@ -150,6 +150,8 @@ These above named MDS damages can be repaired by using the following command::
 If scrub is able to repair the damage, the corresponding entry is automatically
 removed from the damage table.
 
+Note: Scrub with repair flag can idetify the hard link damage but can not repair it.
```

Comment:
nit: scrub invoked with `repair` option can identify an damaged hard link but not repair it.